graph/client.py

```diff
@@ -89,6 +89,9 @@ def parse_arguments(self, args):
         parser = argparse.ArgumentParser(
             description='Client for visualization of SCAP rule evaluation results')
 
+        parser.add_argument('--showFailRules', action="store_true", default=False, help='Show all FAIL rules')
+        parser.add_argument('--showNotSelectedRules', action="store_true", default=False, help="Show not selected rules, but you can't visualized this rules.")   
+        parser.add_argument('--removePassTests', action="store_true", default=False, help='If graph have many nodes in graph, you can remove passing tests for better orientation.(not implemented)')
```

Comment:
Too long. What about: Removes passing tests from the graph, which improves orientation in large graphs.